lessons/ro/chapter_9.yaml

```diff
@@ -158,7 +158,7 @@
     să știe de unde să înceapă.
 
 
-    Ferris spune: "Fiți un rustacean bun și ajutați alți crabi cu un
+    Ferris spune: "Fiți un rustăcean bun și ajutați alți crabi cu un
```

Comment:
I think 'rustacean' sounds better in this context that 'rustăcean'.
Refer to the [declension of the word 'crustaceu'](https://dexonline.ro/definitie/crustaceu/paradigma), from which the word 'rustacean' is derived.